syntaxes/templ.tmLanguage.json

```diff
@@ -436,23 +436,11 @@
         }
       ]
     },
-    "empty-import-expression": {
-      "match": "@(?:[A-z_][A-z_0-9]*\\.)?[A-z_][A-z_0-9]*\\(.*\\)\\s*$",
-      "captures": {
-        "0": {
-          "patterns": [
-            {
-              "include": "#import-expression-start"
-            }
-          ]
-        }
-      }
-    },
```

Comment:
I think this got added back somewhere in my last PRs, whoops. It's not actually getting used and I verified that import-expressions are all working as before.